### DISCUSSION TITLE
export motion frames from fulltime recording

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#3905] [General]
Hello

is post export of motion detected frames from a fulltime recording possible (e.g. after a review process) ?

Is this a munual or automatical process?

Thank's!

Joe

**Comment by NickM-27** [comment]
Can you elaborate on what you're asking, not 100% sure I understand.

**Comment by sudo-joe** [comment]
Ok will try...

lets say during a manual review (preferably at higher speed rates if possible) of the whole recodings  (e.g. made by Frigate made some hours before), motion frames are seen/detected, is it possible to have Frigate that motion frames exported?

**Reply by NickM-27** [reply]
Thanks, not currently possible in frigate UI but it is on the roadmap

**Reply by sudo-joe** [reply]
> Thanks, not currently possible in frigate UI but it is on the roadmap

Will this pro feature included rather in the next few month or in a year or more?

**Comment by sudo-joe** [comment]
Vey good to know :) Is it already today possible to habe Frigate saved/exported (only)the motion detected frames?

**Reply by NickM-27** [reply]
yes, see https://docs.frigate.video/configuration/record#what-do-the-different-retain-modes-mean

**Comment by sudo-joe** [comment]
-->Ok will try...
-->Lets say during a manual review of the recording (preferably at higher speed rates if possible) 
-->motion frames are seen/detected (by the examiner), is it possible to have Frigate that motion 
-> frames (which were manually tagged during the review) exported?

->Thanks, not currently possible in frigate UI but it is on the roadmap

Very happy to hear that this feature is on the roadmap. Once available it will give Frigate a big competitive advantage!!
